crates/pixi_command_dispatcher/tests/integration/main.rs

```diff
@@ -42,6 +46,23 @@ fn tool_platform() -> (Platform, Vec<GenericVirtualPackage>) {
     (platform, virtual_packages)
 }
 
+fn manifest_dir() -> &'static Path {
+    Path::new(env!("CARGO_MANIFEST_DIR"))
+}
+
+fn repository_dir() -> PathBuf {
+    manifest_dir().join("../..")
+}
```

Comment:
You can get workspace dir using this method https://github.com/rust-lang/cargo/issues/3946#issuecomment-973132993.

Reply:
Nice trick! Added it!